content/WebAuthn/Supporting_U2F_or_FIDO2_Security_Keys_on_iOS_or_iPadOS/Known_Issues.adoc

```diff
@@ -0,0 +1,33 @@
+== Known Issues (when using Safari)
+
+*(Updated Dec 20, 2019) This known issues section will be updated as issues are reported.
```

Comment:
the * here causes a formatting error, if this is meant to be in bold it needs to end with a * where bold is to be removed, it it's meant to be a bullet point it needs a space after the *

Reply:
This sentence was a bullet point in the original doc, but that was unnecessary because there was only a single bullet point. So I have removed it.